src/main/java/net/dv8tion/jda/core/utils/Checks.java

```diff
@@ -20,6 +20,19 @@
 
 public class Checks
 {
+    private static final int maxSnowflakeLength = Long.toUnsignedString(-1L).length();
+
+    public static void isSnowflake(final String snowflake)
+    {
+        isSnowflake(snowflake, snowflake);
+    }
+
+    public static void isSnowflake(final String snowflake, final String message)
+    {
+        notNull(snowflake, message);
+        if (snowflake.length() < 1 || snowflake.length() > maxSnowflakeLength || !Helpers.isNumeric(snowflake))
```

Comment:
Should we hardcode the length to 20 instead? I did this because i thought it might be more readable but that would only be the case if the reader knows about two's complement.